src/containers/App.js

```diff
@@ -149,7 +154,10 @@ export default class App extends React.Component {
         }}
         path={path}
         search={doSearch}
-        opened={leftNavIsOpen}/>}
+        opened={leftNavIsOpen}
+        networkNavAnimation={moveWithMenu}
```

Comment:
Cool. Good to merge?

On Thu, Jul 21, 2016 at 6:14 PM, Lawrence Wang <notifications@github.com>
wrote:

> In src/containers/App.js
> <https://github.com/Hylozoic/hylo-redux/pull/82#discussion_r71814407>:
>
> > @@ -149,7 +154,10 @@ export default class App extends React.Component {
> >          }}
> >          path={path}
> >          search={doSearch}
> > -        opened={leftNavIsOpen}/>}
> > +        opened={leftNavIsOpen}
> > +        networkNavAnimation={moveWithMenu}
>
> ok, sounds good
> … <#m_3192980800597093735_>
> On Thu, Jul 21, 2016 at 5:53 PM Robbie Carlton ***@***.***> wrote: In
> src/containers/App.js <#82 (comment)
> <https://github.com/Hylozoic/hylo-redux/pull/82#discussion_r71812962>>: >
> @@ -149,7 +154,10 @@ export default class App extends React.Component { >
> }} > path={path} > search={doSearch} > - opened={leftNavIsOpen}/>} > +
> opened={leftNavIsOpen} > + networkNavAnimation={moveWithMenu} to have it
> behave like it's part of topNav I'm ending up duplicating way more css and
> now looking at adding the scroll listener stuff from topnav also . The code
> is way simpler if I just pass the animation parameters. — You are receiving
> this because you were mentioned. Reply to this email directly, view it on
> GitHub <
> https://github.com/Hylozoic/hylo-redux/pull/82/files/1d1d924e22f720402bc42f5ea4ec7930b99b3c41#r71812962>,
> or mute the thread <
> https://github.com/notifications/unsubscribe-auth/AAL0g_jVP31hrlg5ho6Aaj-mhMtvc6YCks5qYBR9gaJpZM4JSSi7>
> .
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Hylozoic/hylo-redux/pull/82/files/1d1d924e22f720402bc42f5ea4ec7930b99b3c41#r71814407>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA2Y9CH4ShvuekveEUQO__yoqnYBZhp_ks5qYBl5gaJpZM4JSSi7>
> .
>



-- 
Robbie Carlton

719.358.3482
<robbie@authenticmanprogram.com>